src/open_producten/producttypen/models/validators.py

```diff
@@ -36,3 +38,26 @@ def validate_thema_gepubliceerd_state(hoofd_thema, gepubliceerd, sub_themas=None
 def disallow_hoofd_thema_self_reference(thema, hoofd_thema):
     if thema and hoofd_thema and thema.id == hoofd_thema.id:
         raise ValidationError("Een thema kan niet zijn eigen hoofd thema zijn.")
+
+
+def check_processen_url():
+    if not ExterneVerwijzingConfig.get_solo().processen_url:
+        raise ValidationError(
+            _("De processen url is niet geconfigureerd in de externe verwijzing config")
+        )
```

Comment:
You can change it to be generic:

```suggestion
def check_externe_verwijzing_config_url(field_url):
    if not getattr(ExterneVerwijzingConfig.get_solo(), field_url, ""):
        raise ValidationError(
            _("De {field_url} is niet geconfigureerd in de externe verwijzing config").format(field_url=field_url)
        )
```